package.json

```diff
@@ -32,14 +32,13 @@
     "@types/uuid": "^7.0.2",
     "@types/validator": "^13.1.0",
     "@types/yup": "^0.29.11",
-    "apollo": "^2.31.0",
+    "apollo": "^2.33.4",
     "axios": "^0.21.1",
     "classnames": "^2.2.6",
     "craco-sass-resources-loader": "^1.1.0",
     "detect-browser": "^5.0.0",
     "font-awesome": "^4.7.0",
     "formik": "^2.1.4",
-    "graphql": "^15.4.0",
```

Comment:
Why is graphql removed here and not replaced?

Reply:
It's being pulled in by a few of our other dependencies, so we don't need to list it explicitly. Having it in our app dependencies was causing issues on CI because there were multiple versions installed in the app.

```console
$ yarn why graphql
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "graphql"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "graphql@15.4.0"
info Has been hoisted to "graphql"
info Reasons this module exists
   - Hoisted from "apollo#graphql"
   - Hoisted from "apollo#apollo-language-server#graphql"
info Disk size without dependencies: "3.45MB"
info Disk size with unique dependencies: "3.45MB"
info Disk size with transitive dependencies: "3.45MB"
info Number of shared dependencies: 0
=> Found "graphql-schema-linter#graphql@15.5.0"
info This module exists because "graphql-schema-linter" depends on it.
info Disk size without dependencies: "3.47MB"
info Disk size with unique dependencies: "3.47MB"
info Disk size with transitive dependencies: "3.47MB"
info Number of shared dependencies: 0
✨  Done in 1.20s.
```